src/components/UserInfo.js

```diff
@@ -1,7 +1,8 @@
 export default class UserInfo {
-  constructor({ nameSelector, jobSelector }) {
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._jobElement = document.querySelector(jobSelector);
+    this._avatarSelector = document.querySelector(avatarSelector);
```

Comment:
`    this._avatarSelector`


Word `selector` means a **string** that is used for searching elements in the DOM

* You haven't corrected something https://github.com/marzahlde09/se_project_aroundtheus/pull/2#discussion_r1422028750 



So, you need to call it just `this._avatar`



```js
    this._avatar = document.querySelector(avatarSelector);


```